crime-means-assessment/build.gradle

```diff
@@ -7,6 +7,7 @@ plugins {
     id "org.sonarqube" version "4.0.0.2929"
     id "info.solidsoft.pitest" version "1.9.11"
     id "org.jsonschema2dataclass" version "6.0.0"
+    id "se.thinkcode.cucumber-runner" version "0.0.11"
```

Comment:
Not sure how confident I am in this plugin. Running via JUnit seems to be a good alternative and would eliminate the need for a plugin. There are some good guides here:

- https://thepracticaldeveloper.com/cucumber-tests-spring-boot-dependency-injection/
- https://www.baeldung.com/java-cucumber-gradle#running-using-junit

Reply:
It seems to work fine. Do you have any specific objections?

Reply:
It has been maintained for the last N years, and it's referenced by baeldung.org, so I assume its ok.